components/CSVTable.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react'
+import Table from '@mui/material/Table'
+import TableBody from '@mui/material/TableBody'
+import TableCell from '@mui/material/TableCell'
+import TableContainer from '@mui/material/TableContainer'
+import TableHead from '@mui/material/TableHead'
+import TableRow from '@mui/material/TableRow'
+import Paper from '@mui/material/Paper'
+
+function createData (
+  email: string,
+  subject: string,
+  name: string,
+  company: string
+) {
+  return { email, subject, name, company }
+}
+
+const rows = [
+  createData('example@facebook.com', 'Sponsoring HackBeanpot', 'Danaerys', 'Facebook'),
+  createData('example@google.com', 'Sponsoring HackBeanpot', 'Jaime', 'Google'),
+  createData('example@appfolio.com', 'Sponsoring HackBeanpot', 'Tyrion', 'Appfolio')
+]
+
+export default function CSVTable () {
+  return (
+    <TableContainer component={Paper}>
+      <Table aria-label="simple table">
+        <TableHead>
+          <TableRow>
+          <TableCell align="left">email</TableCell>
+            <TableCell align="left">subject</TableCell>
+            <TableCell align="left">name</TableCell>
+            <TableCell align="left">company</TableCell>
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {rows.map((row) => (
+            <TableRow
+              key={row.name}
+              sx={{ '&:last-child td, &:last-child th': { border: 0 } }}
+            >
+              <TableCell align="left">{row.email}</TableCell>
+              <TableCell align="left">{row.subject}</TableCell>
+              <TableCell align="left">{row.name}</TableCell>
+              <TableCell align="left">{row.company}</TableCell>
```

Comment:
Once you pass in the rows as a prop to CSVTable, you can do a map here as well to create these TableCells. Take a look at how this is done in emailSender.tsx with Object.values
<img width="606" alt="Screen Shot 2022-07-01 at 3 45 19 PM" src="https://user-images.githubusercontent.com/59738880/176959758-1ee4e286-fcaa-4035-af68-f3cd39407d50.png">